benchbuild/utils/actions.py

```diff
@@ -549,11 +553,11 @@ def __str__(self, indent=0):
 
 
 class CleanExtra(Step):
-    NAME = "CLEAN EXTRA"
-    DESCRIPTION = "Cleans the extra directories."
+    NAME: str = "CLEAN EXTRA"
+    DESCRIPTION: str = "Cleans the extra directories."
 
     @notify_step_begin_end
-    def __call__(self):
+    def __call__(self) -> StepResult:
```

Comment:
R1710 (inconsistent-return-statements): Either all return statements in a function should return an expression, or none of them should.  <!-- Automatically posted by Lintly -->